Makefile

```diff
@@ -364,6 +373,10 @@ ifneq ($(BUILD),local)
 	@echo "$@ docker container is ready to use"
 endif
 
+ifeq ($(findstring adam,$(HV)),adam)
+ADAM_SSH_KEY:=$(shell rm -f $(CONF_DIR)/ssh.keys && ssh-keygen -P "" -f $(CONF_DIR)/ssh.keys && mv $(CONF_DIR)/ssh.keys.pub $(CONF_DIR)/authorized_keys)
```

Comment:
Can you change this to an actual makefile target, so we can regenerate these keys with a `make` command?

Reply:
That's a good point @deitch -- definitely didn't mean to leave it like that -- it was meant like a quick hack.

Reply:
OK thanks.

Reply:
Also, you need to add `conf/ssh.keys` to `.gitignore`

Reply:
maybe `conf/authorized_keys` also

Reply:
The problem is that `conf/authorized_keys` is checked into git -- we may need to figure out how we deal with files like that -- but I'll punt for now.